python/pyspark/sql/column.py

```diff
@@ -320,10 +323,13 @@ def alias(self, *alias):
     def cast(self, dataType):
         """ Convert the column into type ``dataType``.
 
+        >>> df = spark.createDataFrame([(2, 'Alice'), (5, 'Bob')], ['age', 'name'])
         >>> df.select(df.age.cast("string").alias('ages')).collect()
-        [Row(ages=u'2'), Row(ages=u'5')]
+        [Row(ages='2'), Row(ages='5')]
```

Comment:
If I remember correctly, this test breaks the tests in Python 2.x. Let's use unicodes here.